pipelines/folioRancher/folioTestingTools/folioSchemaCompare.groovy

```diff
@@ -0,0 +1,221 @@
+#!groovy
+import hudson.util.Secret
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.time.*
+import org.folio.rest.GitHubUtility
+import org.folio.Constants
+import groovy.json.JsonSlurperClassic
+
+@Library('pipelines-shared-library@RANCHER-838') _
+
+String getOkapiVersion(folio_repository, folio_branch) {
+    def installJson = new URL('https://raw.githubusercontent.com/folio-org/' + folio_repository + '/' + folio_branch + '/install.json').openConnection()
+    if (installJson.getResponseCode().equals(200)) {
+        String okapi = new JsonSlurperClassic().parseText(installJson.getInputStream().getText())*.id.find{it ==~ /okapi-.*/}
+        if(okapi){
+            return okapi - 'okapi-'
+        } else {
+            error("Can't get okapi version from install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+        }
+    }
+    error("There is no install.json in ${folio_branch} branch of ${folio_repository} repository!" )
+}
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20')),
+    disableConcurrentBuilds(),
+    parameters([
+        jobsParameters.refreshParameters(),
+        jobsParameters.repository(),
```

Comment:
jobParameters is deprecated please use folioParameters

Reply:
<img width="561" alt="image" src="https://github.com/folio-org/pipelines-shared-library/assets/138441317/780057ac-37d0-4a4d-bab1-c227c5a9a095">